src/interface/inference.py

```diff
@@ -0,0 +1,180 @@
+'''Run a REPL or corpus-level inference
+'''
+# pylint: disable=no-member
+import argparse
+import numpy as np
+import os
+import pandas as pd
+import sys
+
+import logging as log
+from tqdm import tqdm
+
+import torch
+
+from src.tasks.tasks import process_sentence, sentence_to_text_field
+from src.utils import config
+from src.utils.utils import load_model_state, check_arg_name
+from src.preprocess import build_tasks, build_indexers
+from src.models import build_model
+
+from allennlp.data import Vocabulary
+from allennlp.data.dataset import Batch
+from allennlp.data import Instance
+from allennlp.nn.util import move_to_device
+
+log.basicConfig(format='%(asctime)s: %(message)s',
+                datefmt='%m/%d %I:%M:%S %p', level=log.INFO)
+
+
+def handle_arguments(cl_arguments):
+    parser = argparse.ArgumentParser(description='')
+    # Configuration files
+    parser.add_argument('--config_file', '-c', type=str, nargs="+",
```

Comment:
Is there an option to load an existing run, instead of relying on the configs to match?

Reply:
What would that involve?

Reply:
We can also use the `params.conf` generated for each run.